### DISCUSSION TITLE
在dolphinscheduler3.2.2中配置http任务如何传入accesstoken

### DISCUSSION THREAD

**Discussion by 966699** [apache/dolphinscheduler#16871] [Q&A]
version：dolphinscheduler3.2.2
我大致的实现思路：通过accesstoken接口获取accesstoken->传入到数据接口地址获得数据，
在配置数据地址接口地址请求时，传入的参数无效，无法获取到信息。在postman中能够正确获取到数据，另外我不太清楚我在dolphinscheduler中这样设置accesstoken是否正确？
![postman调试信息](https://github.com/user-attachments/assets/3260754c-f459-4d54-bec7-def680c5d74d)
![http运行日志](https://github.com/user-attachments/assets/233fb330-814e-4550-88fa-6d417bfe9d13)
![http任务配置](https://github.com/user-attachments/assets/e3dc9620-5f1e-44bc-ab57-23390734cb59)
![接口地址及参数](https://github.com/user-attachments/assets/5da8a652-153f-4efe-b643-c88a3c5d6ae4)

**Comment by Msq2582** [comment]
没看到报错日志

**Reply by 966699** [reply]
有日志图片，就是接口返回的错误，可能图片没有加载出来
